database/database.go

```diff
@@ -167,9 +167,8 @@ func (db DatastoreBacked) StatesForDomains(domains []string) (states []DomainSta
 
 	getDomainStates := func(keys []*datastore.Key) ([]DomainState, error) {
 		domainStates := make([]DomainState, len(keys))
-		getErr := client.GetMulti(c, keys, domainStates)
-		if getErr != nil {
-			return nil, getErr
+		if dsErr := client.GetMulti(c, keys, domainStates); dsErr != nil {
```

Comment:
minor styling nit: this can just be called err, since it's only used here and there is no risk of confusing with another err.

Reply:
Done